### ISSUE TITLE
Plan lekcji w kafelku na stronie głównej i w widgecie jest trochę nieczytelny przy przeniesionych lekcjach

### ISSUE THREAD

**Issue by tadekdudek** [szkolny-eu/szkolny-android#169]
W wyżej wspomnianych miejscach plan lekcji jest trochę nieczytelny. Lekcja odwołana i przeniesiona na tą samą godzinę są w dwóch wierszach. Dużo bardziej przejrzyście byłoby, gdyby to wszystko wcisnąć w jeden wiersz.
![IMG_20230109_212332](https://user-images.githubusercontent.com/99012892/211403234-97c02ab5-0ee0-4c14-99b1-f3d1db89f59e.jpg)

![IMG_20230109_212628](https://user-images.githubusercontent.com/99012892/211403244-1808a04e-4d6d-4e86-9c20-996620344c56.jpg)

**Comment by kuba2k2** [comment]
Oj, tak to się nie da

**Comment by tadekdudek** [comment]
Można też pokazać tylko lekcję przeniesioną i to, że jest przeniesiona, a zamiast jakiej lekcji jest to w popupie

**Comment by kuba2k2** [comment]
Chodzi o to że lekcje przeniesione to okropny bałagan, czasami nawet zdarza się że lekcja zastępowana nie jest odwołana... albo jest kilka na raz. Albo w ogóle nawet przedmioty się nie zgadzają.

**Comment by hrw** [comment]
A może zrobić tak:

11:50 Język polski (~~Edukacja dla bezpieczeństwa~~)
12:45 ~~Język polski~~

**Comment by kuba2k2** [comment]
Nie, to kwestia dziennika (lub warstwy API) a nie prezentacji (UI). Obiekt "lekcja" ma pola oznaczające przedmiot który "miał być" (w przypadku lekcji odwołanych, zastępstw, lub przeniesień jako źródło) oraz przedmiot który "jest" (w przypadku lekcji normalnych, zastępstw, lub przeniesień jako cel).

W tym przypadku dziennik podał nieprawidłowe dane (bo nauczyciel je tak wpisał) lub API nie wie co robi. Nie planujemy już naprawiać błędów nauczycieli, ani tak beznadziejnego systemu jakim jest Librus.
